src/course-unit/constants.js

```diff
@@ -84,3 +86,18 @@ export const PUBLISH_TYPES = {
   discardChanges: 'discard_changes',
   makePublic: 'make_public',
 };
+
+export const getXBlockSupportMessages = (intl) => ({
+  fs: { // Fully supported
```

Comment:
[question] is it not better to provide full key name instead of abbreviation? Like `fullySupported` to make it more clear and avoid comment then.

Reply:
This keys we receive from the API and here we just map them